cve_bin_tool/checkers/sqlite.py

```diff
@@ -166,7 +196,6 @@ def get_version_map():
             version_map.append([last_version, id_match.group(1)])
             print('        [\"' + last_version + '", "' + id_match.group(1) + '"],')
 
-
```

Comment:
Please remove the extra line.  (and if this is being auto-done by pycharm, maybe change your pycharm settings?)

Reply:
When I open pycharm, it shows grey lines . To stop that I used to press the reformatting shortcut.  I will stop doing that.